res/values/strings.xml

```diff
@@ -20,6 +20,7 @@
 -->
 <resources>
```

Comment:
This is like a file for constants to be reused?

Reply:
Pretty much, you can put various types there, like strings, colors, dimensions etc. But here we only keep the strings used in the app, sort of like i18n.